### DISCUSSION TITLE
【社区奖励】WTF贡献者zkSync奖励

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#755] [Announcements]
WTF Academy被zkSync列为外部贡献项目，这意味着我们有50位贡献者可以获得额外的zkSync $zk 代币空投（领取时间待通知）。

我参考了[龙年贡献者红包名单](https://github.com/AmazingAng/WTF-Solidity/discussions/626)和今年突出的贡献者（主要是 WTF zkSync 和 WTF zk 教程），得到了最终获奖名单。

感谢zkSync对WTF Academy的认可，感谢WTF Academy贡献者的不断贡献。

这次没收到红包的贡献者也不要灰心，努力终有回报，我们会不定期对贡献者进行回馈！

名单如下（GitHub Username）：

| GitHub                                                                                       |
| -------------------------------------------------------------------------------------------- |
| AmazingAng                                                                                   |
| reborn-sama                                                                                  |
| tankxu                                                                                       |
| 0xb00lean                                                                                    |
| stoneworld                                                                                   |
| 0xDjangoMo                                                                                   |
| chongqiangchen                                                                               |
| 22Xy                                                                                         |
| Glacier-Luo                                                                                  |
| CallOrRet                             |
| ckjxxf
| murrayee                                                                                     |
| ariesjia                                            |
| seasidejuvenile666                                                                           |
| buttonwild                                                                                   |
| Mosamorphing                                                                                 |
| JustinAsdz                                                                                   |
| 0xKaso                                                                                       |
| 0x0077                                                                                       |
| EasyChris                                                                                    |
| flyq                                                                                         |
| 0xAWM                                                                                        |
| SunWeb3Sec                                                                                   |
| XdpCs                                                                                        |
| hubingliang                                                                                  |
| sdfgsdfsdf                                                                                   |
| pcdotfan                                                                                     |
| 0xkookoo                                                                                     |
| sugarforever                                                                                 |
| kongtaoxing                                                                                  |
| JangoCCC                                                                                     |
| hattiepwvi                                                                                   |
| Big-Aaron                                                                                    |
| cxp-13                                                                                       |
| magiconch                                                                                    |
| yanboishere                                                                                  |
| BrianSeong99                                                                                 |
| zklim                                                                                        |
| spn21                                                                                        |
| Azleal                                                                                        |
| icy101u                                                                                      |
| JackLee                                                                                      |
| jes16jupyter                                                                                 |
| jeasonstudio                                                                                 |
| Xor0v0                                                                                       |
| Erin1314                                                                                     |
| miles-six                                                                                    |
| Zhengqian8958                                                                                |
| yutingzhao1991                                                                               |
| cjh20000613 |

**Comment by yanboishere** [comment]
感恩的心

**Reply by fuchanghai** [reply]
感谢有你

**Reply by lcy101u** [reply]
伴我一生

**Reply by yanboishere** [reply]
_让我有勇气做我自己～_ 😂

**Reply by jeka0203** [reply]
Wow. Cool
